pkg/dpll/dpll.go

```diff
@@ -43,6 +50,25 @@ type DpllConfig struct {
 	dependingState         []event.EventSource
 	exitCh                 chan struct{}
 	ticker                 *time.Ticker
+	// DPLL netlink connection pointer. If 'nil', use sysfs
+	conn *nl.Conn
+	// We need to keep latest DPLL status values, since Netlink device
+	// change indications don't contain all the status fields, but
+	// only the changed one(s)
```

Comment:
Make this comment more clear - there are two dplls, and only one can change at a time. Hence need to store the state